ajax/libs/lottie/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "lottie",
```

Comment:
Please change name of the library as in lib package.json is: https://github.com/airbnb/lottie-web/blob/1f5b8efd40c51a6ae5cfbe0162a1b4f6b0e89872/package.json#L2